src/ray/rpc/grpc_server.cc

```diff
@@ -116,6 +116,7 @@ void GrpcServer::PollEventsFromCompletionQueue(int index) {
   while (cqs_[index]->Next(&tag, &ok)) {
     auto *server_call = static_cast<ServerCall *>(tag);
     bool delete_call = false;
+    bool need_new_call = false;
```

Comment:
Document that a new call is needed after the server sends a reply, no matter the reply is successful or failed.